src/docfx/SubCommands/BuildCommand.cs

```diff
@@ -307,45 +301,30 @@ private static void MergeGitContributeToConfig(BuildJsonConfig config)
             }
         }
 
-        internal static Dictionary<string, FileMetadataPairs> GetFileMetadataFromOption(ListWithStringFallback fileMetadataFilePaths, Dictionary<string, FileMetadataPairs> fileMetadataFromConfig)
```

Comment:
>GetFileMetadataFromOption [](start = 62, length = 25)

could the method and GetGlobalMetadataFromOption be extracted to one method? seems the logic is same